lib/utils/identifierMapping.js

```diff
@@ -180,13 +180,12 @@ function knexIdentifierMappers({ parse, format, idSeparator = ':' } = {}) {
 
     postProcessResponse(result) {
       if (Array.isArray(result)) {
-        const output = new Array(result.length);
-
-        for (let i = 0, l = result.length; i < l; ++i) {
-          output[i] = parseKeys(result[i]);
-        }
-
-        return output;
+        return result.map(parseKeys);
+      } else if (result.rows) {
```

Comment:
What if the `result` is a single record (row) and the row has a column called `rows`? Or can `result` ever be an object here?

Reply:
@koskimas: I've put in some test cases for that and it seems fine. 
I had a test for a postgres jsonb column named "rows" with snake_cased keys just to ensure that `camelCaseKeys` didn't parse into that when returning. It didn't, but the test took a lot of setup nested within the `isPostgres()`. It just seemed like too much. I added a unit test for `camelCaseKeys()` directly which I think gets the job done.

Reply:
Also `.raw` query responses are different depending on db driver. Is there something that should be fixed for other drivers too?

Reply:
I started looking into the raw response for MySQL and Sqlite and got it working but the complexity of handling every raw response for the different DBs is adding up. I don't have access to MSSQL or Oracle so I can't do anything there.

If other people think having this in Objection for raw queries is worth it, I think the way to do it would be separate functions for each DB's raw response shape. People can then opt-in to just the function they need for their DB.

Let me know if that's something you would want.

For now, I'm just going to use my own knex mapper that works for what I need.